Makefile

```diff
@@ -40,6 +40,6 @@ start:
 build: check-env clone
 	@ printf "\n[${GREEN} Generating environment variables... ${NO_COLOR}]\n"
 	cd ${SDX_HOME}/sdx-common && pip install .; cd -
-	cd ${SDX_HOME}/sdx-ops && ${PYTHON3} sdx/ops/configure.py --env > ${SDX_HOME}/sdx-compose/env/private.env ;cd -
+	cd ${SDX_HOME}/sdx-ops && python sdx/ops/configure.py --env > ${SDX_HOME}/sdx-compose/env/private.env ;cd -
```

Comment:
But, to be honest, I don't even know why we are generating keys. Don't we have test keys in compose?